api/column_family.cc

```diff
@@ -6,9 +6,11 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 
+#include <fmt/ranges.h>
 #include "column_family.hh"
 #include "api/api.hh"
 #include "api/api-doc/column_family.json.hh"
+#include "api/api-doc/storage_service.json.hh"
```

Comment:
But, these endpoints are still live under `/storage_service/` prefix, no?

Reply:
Yes, the path they are visible under is not changed

Reply:
@denesb , does your concern still stand?

Reply:
I think keeping related code together is more important than repeating the mistake of the API endpoint placement.